src/Dapr.Client/DaprClient.cs

```diff
@@ -206,15 +208,16 @@ public abstract class DaprClient
         /// <param name="storeName">The name of the state store.</param>
         /// <param name="key">The state key.</param>
         /// <param name="consistencyMode">The consistency mode <see cref="ConsistencyMode" />.</param>
+        /// <param name="metadata">An key/value pair that may be consumed by the state store.  This is dependent on the type of state store used.</param>
```

Comment:
nit: "A" key/value pair :)

Reply:
All mentions in DaprClient in "an key/value" is fixed.